bin/update-dockerfiles.sh

```diff
@@ -52,13 +54,15 @@ update_generic_linux() {
     jdk_version=$(echo ${CORRETTO_VERSION} | cut -d'.' -f1-3)
     jdk_build=$(echo ${CORRETTO_VERSION} | cut -d'.' -f4)
     corretto_version=$(echo ${CORRETTO_VERSION} | cut -d'.' -f5)
-    ${SED} "s/ARG version=.*/ARG version=${jdk_version}.${jdk_build}-${corretto_version}/g" ./${MAJOR_RELEASE}/jdk/al2-generic/Dockerfile
+    if [[ "${AL2_VERSIONS[*]}" =~ ${MAJOR_RELEASE} ]]; then
+        ${SED} "s/ARG version=.*/ARG version=${jdk_version}.${jdk_build}-${corretto_version}/g" ./${MAJOR_RELEASE}/jdk/al2-generic/Dockerfile
+    fi
     if [[ -f ./${MAJOR_RELEASE}/jdk/al2023-generic/Dockerfile ]]
```

Comment:
We can remove this as we don't think al2023 with generic linux packages anymore.